src/ImageSharp/Dithering/ErrorDiffusion/IErrorDiffuser.cs

```diff
@@ -0,0 +1,34 @@
+﻿// <copyright file="IErrorDiffuser.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Dithering
+{
+    using System;
+
+    /// <summary>
+    /// Encapsulates properties and methods required to perfom diffused error dithering on an image.
+    /// </summary>
+    public interface IErrorDiffuser
+    {
+        /// <summary>
+        /// Gets the dithering matrix
+        /// </summary>
+        byte[][] Matrix { get; }
```

Comment:
@JimBobSquarePants why are you using `byte`-s here? I think perf would be much better with floats.
I also keep promoting [my idea](https://gist.github.com/antonfirsov/6c11a2f0a1048e8bcca455c072d14810) to flatten these arrays.